transport/http2_server.go

```diff
@@ -142,16 +145,24 @@ func newHTTP2Server(conn net.Conn, config *ServerConfig) (_ ServerTransport, err
 			Val: maxStreams,
 		})
 	}
-	if initialWindowSize != defaultWindowSize {
+	iwz := int32(initialWindowSize)
+	if config.InitialWindowSize >= defaultWindowSize {
```

Comment:
The send quota on both client and server for both stream and connection takes the default value of defaultWindowSize. Therefore, that can be our lower cap. Is there an issue if it's smaller than initialWindowSize?